project/build.scala

```diff
@@ -0,0 +1,83 @@
+import sbt.Keys._
+import sbt._
+
```

Comment:
I thought you wanted the backend to be in a separate repo? If so the way to handle it should be via an artifact (i.e. managed dependency), no?